src/storage/disk_manager.c

```diff
@@ -108,10 +109,10 @@ struct disk_recv_change_creation
   char vol_fullname[1];		/* Actually more than one */
 };
 
-typedef struct disk_recv_init_pages_info DISK_RECV_INIT_PAGES_INFO;
-struct disk_recv_init_pages_info
-{				/* Recovery for volume page init */
-  INT32 start_pageid;
+/* recovery data for volume expand */
+typedef struct disk_recv_data_volume_expand DISK_RECV_DATA_VOLUME_EXPAND;
+struct disk_recv_data_volume_expand
```

Comment:
Since we may have to also patch 10.1, I'd like to propose to keep log data as it was. 
We may delete unused `start_pageid` when we change log compatibility number.